pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -502,6 +516,7 @@
   module_grade_2_5: "2-5"
   module_grade_2_8: "2-8"
   module_grade_2_12: "2-12"
+  module_grade_2_13: "2-13"
```

Comment:
Is this for a grade range of 2-13? And will we use this somewhere else?

Reply:
Oh great catch! This is a typo, I'll remove it.